app/javascript/stylesheets/gobierto-data.scss

```diff
@@ -359,27 +423,37 @@
     margin-top: 0;
     font-family: monospace;
     font-size: 12px;
-    color: #666;
+    color: var(--grey-text);
     position: sticky;
     top: 0;
   }
 
   &-btn-download-data {
     border: 1px solid rgba(var(--color-base-string), .2);
-    padding: 1rem;
+    padding: .5rem;
     font-size: 1rem;
     font-family: $font_text;
     font-weight: 600;
     min-width: 250px;
-    margin: 1rem 1rem .5rem .25rem;
+    margin: 0 0 .5rem 0;
     position: relative;
     text-transform: unset;
     vertical-align: top;
   }
 
+  &-summary-header-btns {
+    margin-bottom: 1rem;
+    width: 100%;
+
+    a.gobierto-data-btn-download-data {
```

Comment:
Bad practice. Or classes or selectors, not both. Fix the next one, also

Reply:
Done.